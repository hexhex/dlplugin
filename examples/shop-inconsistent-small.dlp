% -*- prolog -*-

#namespace("shop","http://www.kr.tuwien.ac.at/staff/roman/shop#").

minusShop("shop:Shop",S) :- &dlC["shop.owl",a,b,c,d,"shop:Shop"](S).

consistent :- &dlConsistent["shop.owl",a,minusShop,c,d].
inconsistent :- not &dlConsistent["shop.owl",a,minusShop,c,d].

%shop(S) :- &dlC["shop.owl",a,minusShop,c,d,"shop:Shop"](S).
