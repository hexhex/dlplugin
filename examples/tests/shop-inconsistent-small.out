{inconsistent, minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s1>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s2>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s3>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s4>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s5>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s6>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s7>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s9>"), minusShop("<http://www.kr.tuwien.ac.at/staff/roman/shop#Shop>","<http://www.kr.tuwien.ac.at/staff/roman/shop#s10>")}
